node/src/actors/chain_manager/mining.rs

```diff
@@ -136,18 +135,14 @@ impl ChainManager {
             // FIXME: After Reputation Merkelitation, only the ARS Members from the previous consolidated
             // Superblock will be added, to avoid include addresses that could be wrong later by
             // block reorganization
```

Comment:
Is this fixed now? We use `last_ars` to create the superblock, but is `last_ars` the ARS from the previous superblock?

Reply:
It looks like `last_ars` is the ARS from the previous block, so that's not implemented yet, but it should be easy, right? Just add a `ars_from_last_superblock` field to `ChainState`.

Reply:
I think it is fixed, the`'is_valid` method checks against the `previous_ars_identities` in superblock.rs and I think that is the method checked to see if you need to forward right? If that is the case then `previous_ars_identities` is now the previous superblock ARS so it should be fixed